webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -38,36 +38,39 @@ class SubscriptionsPage extends Component {
   }
 
   static getDerivedStateFromProps(nextProps, prevState) {
-    const nextTaskId = nextProps.tasks[0] && nextProps.tasks[0].id;
+    const { tasks = [] } = nextProps;
```

Comment:
If you'll remove this empty array default value, will it be undefined?  especially when an empty array is defined as the default state? seems very odd to me.

Reply:
Somehow it is happen when I run `centos7-katello-bats-ci` but I couldn't reproduce it locally.

Reply:
Same problem with `centos7-katello-nightly`